modules/nw-infw-operator-cr.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-node-firewall-operator.adoc
+
+:_content-type: CONCEPT
+[id="nw-infw-operator-cr_{context}"]
+= Ingress Node Firewall Operator
+
+The Ingress Node Firewall Operator provides ingress firewall rules at a node level by deploying the daemon set to nodes you specify and manage in the firewall configurations. To deploy the daemon set, you create an `IngressNodeFirewallConfig` custom resource (CR). The Operator applies the `IngressNodeFirewallConfig` CR to create ingress node firewall daemon set `daemon`, which run on all nodes that match the `nodeSelector`.
+
+You configure `rules` of the `IngressNodeFirewall` CR and apply them to clusters using the `nodeSelector` and setting values to "true".
```

Comment:
```suggestion
You configure `rules` of the `IngressNodeFirewall` CR and apply them to clusters using the `nodeSelector` object and setting values to `true`.
```
Should `true` be in monospace or regular quotes?

Reply:
regular, I _think_